tests/by-util/test_split.rs

```diff
@@ -156,3 +165,59 @@ fn test_split_additional_suffix() {
     assert_eq!(glob.count(), 2);
     assert_eq!(glob.collate(), at.read(name).into_bytes());
 }
+
+#[test]
+#[cfg(unix)]
```

Comment:
One last request, could you please add a comment to explain why Windows doesn't work?

Reply:
sure... redundancy is always welcome